src/components/forms/event-category-form.js

```diff
@@ -302,6 +331,43 @@ class EventCategoryForm extends React.Component {
                         </div>
                     </div>
                 </Panel>
+    
+                {entity.id !== 0 &&
```

Comment:
i do think that we need to add the condition that parent is null here  to show this UI portion ?